### PR TITLE
Fix/ set default backgroundColor for Modal

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Modal - set default backgroundColor

## Changelog
Modal - set default backgroundColor

### CODE REVIEW

File: demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default class TextFieldScreen extends Component {
 
   renderPickerContent = () => {
     return (
-      <View bg-white br20 padding-s3 paddingB-60>
+      <View bg-$backgroundDefault br20 padding-s3 paddingB-60>
```
Comment (M-i-k-e-l) [inline demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx:71]: Is this bg still needed?

Comment (lidord-wix) [reply demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx:71]: yes, try to remove it :)

Comment (M-i-k-e-l) [reply demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx:71]: I did, please describe the problem, or maybe I missed something in the reproduction

Comment (lidord-wix) [reply demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx:71]: remove it and open the second picker in the expandableOverlay screen, you'll see that the dialog has no bg

Comment (M-i-k-e-l) [reply demo/src/screens/incubatorScreens/IncubatorExpandableOverlayScreen.tsx:71]: 🤦‍♂️ 
Totally ignored that example...

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**